zgw/objecten-api/src/main/kotlin/com/ritense/objectenapi/service/ZaakObjectService.kt

```diff
@@ -134,8 +141,50 @@ class ZaakObjectService(
         return zakenApiPluginInstance
     }
 
+    fun createObject(objectManagementId: UUID, data: JsonNode) : ZaakObjectDto {
+        val result = objectManagementProvider.getObjectManagementInfo(objectManagementId)
+
+        val objectManagementProviderDTO = ObjectManagementProviderDTO(
```

Comment:
This DTO is only used within this method, so it's not really transferring any data at all. Please use the values where needed, and skip the DTO stuff.

Reply:
Not sure why you marked it as resolved. The DTO is still there.